packages/@aws-cdk/aws-lambda/lib/function-base.ts

```diff
@@ -40,6 +41,11 @@ export interface IFunction extends IResource, ec2.IConnectable, iam.IGrantable {
    */
   readonly isBoundToVpc: boolean;
 
+  /**
+   * The `$LATEST` version of this function.
+   */
+  readonly latestVersion: IVersion;
```

Comment:
Is there no importedfunction that needs this as well?

Reply:
It seems not - it must extend `FunctionBase`, otherwise it wouldn't build for it does not implement `IFunction` correctly...